### PR TITLE
Verify the browser window and webview urls in a basic fashion.

### PR DESCRIPTION
**Pull request by Mossop**

### CODE REVIEW

**Comment by jsantell** [comment]
nice

**Comment by Mossop** [comment]
Still pretty mediocre but at least it verifies the presence and location of the webview. Look good @ncalexan?

File: test/webdriver/test-basic.js [hunk 1]
```diff
@@ -14,18 +14,42 @@ describe('application launch', function() {
...
+    });
   });
 
-  it('shows an initial window', async function() {
+  it('shows an initial window and tab', async function() {
+    let { value: url } = await this.app.client.url();
+    expect(url.startsWith('file://')).toBe(true);
+    expect(url.endsWith('browser.html')).toBe(true);
+
     const count = await this.app.client.getWindowCount();
```
Comment (ncalexan) [inline test/webdriver/test-basic.js]: Worth a comment explaining this?  You and I know, and there's clues above -- but this is *very* surprising, or at least it was to me.

File: test/webdriver/test-basic.js [hunk 2]
```diff
@@ -14,18 +14,42 @@ describe('application launch', function() {
...
+    const { value: handles } = await this.app.client.windowHandles();
+
+    expect(handles.length).toEqual(2);
+    const pos = handles.indexOf(handle);
+    expect(pos).toBeGreaterThanOrEqualTo(0);
+
+    const webviewHandle = handles[1 - pos];
+    await this.app.client.window(webviewHandle);
+
+    ({ value: url } = await this.app.client.url());
```
Comment (ncalexan) [inline test/webdriver/test-basic.js]: Are the parens necessary?  If not, drop them.

**Comment by ncalexan** [comment]
Rock on.

File: test/webdriver/test-basic.js (hunk 2 above)
Comment (Mossop) [inline test/webdriver/test-basic.js]: Yes, lines can't start with {.

**Pull request merged**